r-pkg/R/elasticsearch_eda_funs.R

```diff
@@ -114,3 +114,145 @@ get_counts <- function(field
     
 }
 
+#' @title Get the names and data types of the indexed fields in an index
+#' @name get_fields
+#' @description For a given Elasticsearch index, return the mapping from field name
+#'              to data type for all indexed fields.
+#' @importFrom futile.logger flog.fatal flog.info
+#' @importFrom httr GET content stop_for_status
+#' @importFrom data.table := uniqueN
+#' @param es_host A string identifying an Elasticsearch host. This should be of
+#'                the form \code{[transfer_protocol][hostname]:[port]}. For example,
+#'                \code{'http://myindex.thing.com:9200'}.
+#' @param es_indices A character vector that contains the names of indices for
+#'                   which to get mappings. Default, is \code{'_all'}, which means
+#'                   get the mapping for all indices Names of indices can be
+#'                   treated as regular expressions.
```

Comment:
Could you please fix the punctuation in this param? (remove comma in 128, add period in 129)